heliopy/data/spice.py

```diff
@@ -42,8 +42,12 @@ def make_doc_entry(self):
 
 generic_kernels = [_Kernel('Leap Second Kernel', 'lsk',
                            'https://naif.jpl.nasa.gov/pub/naif/generic_kernels/lsk/naif0012.tls'),
-                   _Kernel('Development Ephemeris', 'planets',
-                           'https://naif.jpl.nasa.gov/pub/naif/generic_kernels/spk/planets/de430.bsp')]
+                   _Kernel('Planet trajectories', 'planet_trajectories',
```

Comment:
I think the test file should be also changed to 'planet_trajectories' from 'planets'

Reply:
Oh yeah, good spot